apps/src/code-studio/progress.js

```diff
@@ -179,6 +179,7 @@ progress.renderCourseProgress = function(scriptData) {
         redirectScriptUrl={scriptData.redirect_script_url}
         versions={scriptData.versions}
         courseName={scriptData.course_name}
+        hideAssignButton={scriptData.hidden}
```

Comment:
Unfortunately I don't think it will be this easy, because the following people still need to be able to assign the course:
* people with hidden script access (internal code.org people only)
* people in pilot experiments (we definitely have CSD piloters right now)
* people in course experiments (may be negotiable)

I would recommend computing a new value like `hide_assign_button` (or `show_assign_button`) and passing it down in the summary. Check out this logic for an idea of how to compute it: https://github.com/code-dot-org/code-dot-org/blob/03f1e98666bd2b2d0f5803d8dbda988681720a32/dashboard/app/models/script.rb#L216-L229

Reply:
Gah! All good points.  Thanks for catching - I'll address.